schemas/view-schema.json

```diff
@@ -194,38 +194,26 @@
                 }
               },
               "readOnly": true
-            }
-          },
-          "readOnly": true
-        },
-        "focus": {
-          "type": "object",
-          "description": "Focus properties",
-          "properties": {
-            "focused_times": {
+            },
+            "in_foreground_times": {
               "type": "array",
-              "description": "List of the time the user had the document focused for browser",
+              "description": "List of the period the user had the view in foreground (focused in the browser)",
               "items": {
                 "type": "object",
-                "description": "Properties of the focused times of the view",
+                "description": "Properties of the foreground period of the view",
                 "required": ["duration", "start"],
                 "properties": {
                   "start": {
                     "type": "integer",
-                    "description": "Timestamp when the user gain focus of the document",
+                    "description": "Relative time in ns when in foreground from the view start",
```

Comment:
```suggestion
                    "description": "Timestamp in ns relative to the view start when the event is in the foreground",
```
maybe?

Reply:
I ended up looking at the definition of a timestamp and it's correct but feels weird to have a timestamp that's not starting at the same time as the UNIX one.
m2c

Reply:
What about having something similar to https://github.com/DataDog/rum-events-format/blob/d0bf4aaa7a95f8720cc9cefcf3d6f723049017bd/schemas/resource-schema.json#L89?
with view start and foreground start here